### DISCUSSION TITLE
Daily Scrum #2

### DISCUSSION THREAD

**Discussion by IrfanZnA** [GIA-003/Kado#21] [General]
meeting hari 2

**Comment by HudaGates** [comment]
Untuk hari ini saya telah menyelesaikan kado dan dekorasinya bisa diliat pada tugasku

**Reply by IrfanZnA** [reply]
OK

**Comment by naufalfadhlullah** [comment]
Aku udah bikin sketsa akhir

**Comment by IrfanZnA** [comment]
Berarti kita besok membuat kartu ucapan dan estimasi biaya nah itu untuk mengakhiri proses development.
